app/models/school_group.rb

```diff
@@ -28,4 +28,12 @@ def county
   def postcode
     gias_data['Group Contact Postcode']
   end
+
+  def region
+    Region.find_by(id: '847bf2e8-ef69-4fb6-9fba-8ae624242e86')
```

Comment:
should there be a hardcoded uuid here? id expect to see it in a test but in an actual app script?

Reply:
Getting rid of this now!